.mergify.yml

```diff
@@ -61,6 +61,19 @@ pull_request_rules:
         - or:
           - -files~=(\.rs|Cargo\.toml|Cargo\.lock)$
           - check-success=clippy (macos-latest)
+        - or:
+          - -files~=(\.rs|Cargo\.toml|Cargo\.lock|cargo-build-bpf|cargo-test-bpf|cargo-build-sbf|cargo-test-sbf)$
```

Comment:
Should this also include `ci/downstream-projects/run-spl.sh` and `.github/workflows/downstream-project-spl.yml` like the GHA downstream-project-spl config?

Reply:
thank you for bringing this up. I ignored them intentionally at the beginning but I changed my mind haha. 😂 (updated: [9d2df67](https://github.com/solana-labs/solana/pull/34326/commits/9d2df6788f3c635d651a4bea9ebaff6f63a6c679))